api/workAllocation/index.spec.ts

```diff
@@ -4,9 +4,9 @@ import 'mocha';
 import * as sinon from 'sinon';
 import * as sinonChai from 'sinon-chai';
 import { mockReq, mockRes } from 'sinon-express-mock';
-
-import { baseWorkAllocationTaskUrl, getTask, postTaskAction, searchTask } from '.';
+import { baseWorkAllocationTaskUrl, getTask, postTaskAction, searchTask, getTypesOfWork } from '.';
 import { http } from '../lib/http';
+import { RE_ALLOCATE, REMOVE_ALLOCATE } from './constants/actions';
 import { mockTasks } from './taskTestData.spec';
```

Comment:
Minor comment - import ordering wrong here

Reply:
explained above comment why we should not fine-tune code while doing removing MVP .